core/modules/image/image.field.inc

```diff
@@ -85,6 +85,10 @@ function image_field_instance_settings_form($field, $instance) {
   // Use the file field instance settings form as a basis.
   $form = file_field_instance_settings_form($field, $instance);
 
+  $form = array(
+    '#element_validate' => array('image_field_settings_form_validate'),
+  );
+
```

Comment:
Instead of replacing all of `$form`, you should append the additional element validate:

```
$form['#element_validate'] = isset($form['#element_validate']) : $form['#element_validate'] : array();
$form['#element_validate'][] = 'image_field_settings_form_validate';
```